parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -297,8 +297,8 @@ disableSystemdResolved() {
     ls -ltr /etc/resolv.conf
     cat /etc/resolv.conf
     UBUNTU_RELEASE=$(lsb_release -r -s)
-    if [[ "${UBUNTU_RELEASE}" == "18.04" || "${UBUNTU_RELEASE}" == "20.04" || "${UBUNTU_RELEASE}" == "22.04" ]]; then
-        echo "Ingorings systemd-resolved query service but using its resolv.conf file"
+    if [[ "${UBUNTU_RELEASE}" == "18.04" || "${UBUNTU_RELEASE}" == "20.04" || "${UBUNTU_RELEASE}" == "22.04" || "${UBUNTU_RELEASE}" == "24.04" ]]; then
```

Comment:
@UtheMan would have been a breaking change here for 1604 if I had done something like:

if isUbuntu; then
  ...
fi